build/.gitmodules.0

```diff
@@ -0,0 +1,51 @@
+[submodule "third_party/zlib"]
```

Comment:
same here. be careful on this. Please don't include this addition of submodules in your final change.